src/microsoft/microsoft.app.js

```diff
@@ -28,11 +28,8 @@ angular.module('Module.microsoft', [
           configurationSelected: true,
         });
       }],
-      translator: ['translator', function (translator) {
-        return translator.load(['microsoft']).then(() => translator);
-      }],
     },
+    translations: ['microsoft'],
   });
-}]).run(['translator', 'MicrosoftOfficeLicenseService', function (translator) {
-  translator.load(['microsoft']);
-}]).constant('MICROSOFT_GUIDE_URLS', {});
+}])
+.constant('MICROSOFT_GUIDE_URLS', {});
```

Comment:
`error  Expected indentation of 2 spaces but found 0  indent`

Reply:
This has been fixed and the build is now successful

Reply:
Ok good :D